packages/components/src/Navbar/Component.purs

```diff
@@ -12,10 +12,9 @@ import Effect (Effect)
 import KSF.Icons (papers)
```

Comment:
I would leave pure style changes to a different pull request, this is large enough even without and they risk merge conflicts.

Reply:
🤔  I don't think I have anything to do with this 😅

Reply:
@kaol What did you exactly mean? I don't know how to act on this :blush:

Reply:
Never mind, now that I looked a bit closer I saw that this was needed since `User` was added to User.purs. I first thought that this just rearranged imports to no effect.